source/documentation/other-topics/modsecurity.md

```diff
@@ -0,0 +1,108 @@
+### ModSecurity - Web Application Firewall
+
+#### Introduction
+
+[ModSecurity](http://www.modsecurity.org/about.html) is an open source, cross platform web application firewall (WAF) developed by Trustwave's SpiderLabs. It has a robust event-based programming language which provides protection from a range of attacks against web applications and allows for HTTP traffic monitoring and logging. 
+
+ModSecurity is a normally third party add-on for nginx, however this is included within the [nginx ingress-controller](https://github.com/kubernetes/ingress-nginx) package for Kubernetes. 
```

Comment:
This section and the bullet list seems more relevant to the cloud platform team than to developers. I imagine the thing they're most interested in is "how do I turn it on for my namespace", as opposed to the details of how we have it set up in the cluster.

Reply:
taken out middle section to keep the section lean